packages/terra-graphs-docs/src/example-datasets/dataObjects/Gantt/dynamicPanningDataset2.js

```diff
@@ -0,0 +1,78 @@
+const scheduled = {
+  path: [
+    {
+      id: 'calendar',
+      d:
+        'M31 5V1.8c0-1-.8-1.8-1.8-1.8h-1.5c-1 0-1.8.8-1.8 1.8V5H11V1.8c0-1-.8-1.8-1.8-1.8H7.8C6.8'
+        + ' 0 6 .8 6 1.8V5H2v33h19.1c-.1-.7-.1-1.3-.1-2v-1H5V17h27v4.6c1-.3 2-.4 3-.5V5h-4z',
+    },
+    {
+      id: 'circle',
+      fill: '#78C346',
+      d:
+        'M36 24c6.6 0 12 5.4 12 12s-5.4 12-12 12-12-5.4-12-12 5.4-12 12-12z',
+    },
+    {
+      id: 'check',
+      fill: '#FFF',
+      d: 'M34.1 42.4l-6.5-6.5 2.1-2.1 4.4 4.3 8.2-8.1 2.1 2.1-10.3 10.3z',
+    },
+  ],
+  options: {
+    x: -7,
+    y: -7,
+    scale: 0.4,
+  },
+};
+
+const dyanmicData = {
+  actions: [
+    {
+      key: 'uid_action_1',
+      values: [
+        new Date(2016, 0, 1, 7, 15).toISOString(),
+        new Date(2016, 0, 1, 8, 15).toISOString(),
+      ],
+    },
+    {
+      key: 'uid_action_2',
+      values: [new Date(2016, 0, 1, 9, 15).toISOString()],
+    },
+  ],
+  tasks: [
+    {
+      key: 'task10',
+      startDate: new Date(2016, 0, 1, 0).toISOString(),
+      endDate: new Date(2016, 0, 1, 13).toISOString(),
+    },
+    {
+      key: 'task11',
+      startDate: new Date(2016, 0, 1, 15).toISOString(),
+      endDate: new Date(2016, 0, 1, 20).toISOString(),
+    },
+  ],
+  events: [
+    {
+      key: 'uid_event_4',
+      shape: scheduled,
+      values: [new Date(2016, 0, 1, 2, 15).toISOString()],
+    },
+  ],
+  activities: [
+    {
+      key: 'activity5',
+      startDate: new Date(2016, 0, 1, 3).toISOString(),
+      endDate: new Date(2016, 0, 1, 6).toISOString(),
+    },
+  ],
+};
```

Comment:
Since we're only using the properties of `dynamicData` and not the object, it would be better to break these into separate variables for actions, tasks, events and activities.

Reply:
Fixed [here](https://github.com/cerner/terra-graphs/pull/229/commits/d8e8b462e04dc4f262cd888740100453f7d02e4a#diff-d7dede05525dc9c2cc2bea44c80c078ea1705cef2b9bde54e66dfe606fc75dc6R28).